go.mod

```diff
@@ -82,11 +82,11 @@ require (
 	github.com/hashicorp/packer-plugin-openstack v1.0.1
```

Comment:
I notice we aren't referencing the `oracle` plugin here, is it normal?

Reply:
Yes, Oracle was removed in a previous release. They no longer support the Solaris builds so we stopped bundling with Packer.

Reply:
Oh yeah now that you mention that I remember this, all good then!